lib/fs/fs.c

```diff
@@ -124,7 +139,7 @@ result_t trn_fs_realpath(const char *path, char **resolved_path) {
 
 	printf("realpath(\"%s\")\n", path);
```

Comment:
remove or put behind `TRN_FS_DEBUG` macro

Reply:
(if you do make a `TRN_FS_DEBUG` macro, use an `if` statement like the `AP_DEBUG` macro for alloc_pages does do we get compiler warnings even when it's disabled)